opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -84,15 +88,20 @@ def get_current_span(self) -> 'Span':
             invalid :class:`.SpanContext`.
         """
 
-
     @contextmanager  # type: ignore
-    def start_span(self, name: str, parent: 'Span') -> typing.Iterator['Span']:
+    def start_span(self,
+                   name: str,
+                   parent: typing.Union['Span', 'SpanContext'] = CURRENT_SPAN
+                   ) -> typing.Iterator['Span']:
         """Context manager for span creation.
 
-        Create a new child of the current span, or create a root span if no
-        current span exists. Start the span and set it as the current span in
+        Create a new span. Start the span and set it as the current span in
         this tracer's context.
 
+        By default the current span will be used as parent, but an explicit
+        parent can also be specified, either a Span or a SpanContext.
+        If the specified value is `None`, the created span will be a root span.
```

Comment:
```suggestion
        If the specified value is ``None``, the created span will be a root span.
```

Sphinx will render single-backticks as italic, double-backticks as code.

Reply:
Unless we merge https://github.com/open-telemetry/opentelemetry-python/pull/40 first!